test/runtests.jl

```diff
@@ -197,3 +196,37 @@ const Pwilcox     = @compat pwilcox.(Rwilcox, 13, 17)
 @compat allEq(Runif,	  qunif.(log.(1 .- Punif), .2, 2, false, true))
 @compat allEq(Rweibull,   qweibull.(log.(1 .- Pweibull), 3, 2, false, true))
 @compat allEq(Rwilcox,	  qwilcox.(log.(1 .- Pwilcox), 13, 17, false, true))
+
+## Test if srand working correctly
+srand(124)
+Rbeta2	  = rbeta(n, .8, 2)
+Rbinom2	  = rbinom(n, 55, pi/16)
```

Comment:
I know tabs are used for alignment elsewhere in the file (unfortunately), but it would be great if you could align your code here with spaces instead. One of the problems with tabs is that they render differently depending on your settings. For example, in the GitHub UI, tabs are 8 spaces wide, so these don't line up at all on GitHub.

Not a dealbreaker or anything, I just think it'd be a good thing to do, if you'd be willing.